src/style.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: local("Avenir")
+    url(D:\IT\Mate\projects\layout_search-bar-airbnb\src\src\fonts\Avenir.ttc);
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+h1 {
+  display: none;
+}
+
+.img {
+  position: absolute;
+  margin: 25px 0 26px 26px;
+}
+
+.imgSmall {
+  position: absolute;
+  height: 11px;
+  width: 11px;
+  margin: 15px 0 16px 13px;
+}
+
+.bigSearch {
+  box-sizing: border-box;
+  margin: 20px 0;
+}
+
+.inputBig {
```

Comment:
use all styles to both inputs to class `input`, use additional classes only for different styles